src/state/CallViewModel.ts

```diff
@@ -464,16 +465,36 @@ export class CallViewModel extends ViewModel {
       },
     );
 
+  /**
+   * Observable for changes to the MatrixRTCSession membership list.
+   *
+   * We do this to ensure that we only listen once to the event and then share internally.
+   */
+  private readonly membershipsChanged$ = fromEvent(
+    this.matrixRTCSession,
+    MatrixRTCSessionEvent.MembershipsChanged,
+  ).pipe(share());
+
+  /**
+   * Observable for changes to the Matrix Room member data.
+   *
+   * We do this to ensure that we only listen once to the event and then share internally.
+   */
+  private readonly roomMembers$ = fromEvent(
+    this.matrixRTCSession.room,
+    RoomStateEvent.Members,
+  ).pipe(share());
```

Comment:
What would be the advantage to use share here instead of context.state()?

Reply:
I don't think we need/want the `distinct()` operator. The same for `shareReplay()` vs `share()`.

It would probably make sense to make this explicit in a docstring for the `roomMember$` and `membershipsChanged$`.